plugin/src/main/java/com/spotify/plugin/dockerfile/MavenRegistryAuthSupplier.java

```diff
@@ -47,9 +58,26 @@ public RegistryAuth authFor(final String imageName) throws DockerException {
     final ImageRef ref = new ImageRef(imageName);
     Server server = settings.getServer(ref.getRegistryName());
     if (server != null) {
+      String password = server.getPassword();
+      boolean encrypted = false;
+      try {
+        encrypted = isEncrypted(password);
+      } catch (PlexusCipherException e) {
+        log.warn("Couldn't determine if Maven server password is encrypted.");
+        log.warn("Assuming Maven server password *is not* encrypted.", e);
+      }
+      try {
+        log.debug("Maven server password is encrypted: {}", encrypted);
```

Comment:
Looks like this line should be inside the following if condition?

Reply:
@davidxia it's in the correct place: it's a debug message that indicates whether the server password is encrypted or not.

Reply:
@davidxia thanks for commenting on this review.  Yes, we're using these changes in a forked version of this repository.  I've also included an Invoker plugin integration test in this pull request.

Reply:
Ah yes. Could you just update the README to indicate this feature has been added then?

Reply:
@davidxia, good catch.  I updated the README [here](https://github.com/spotify/dockerfile-maven/pull/167/commits/d0fb09e622c7e8d788b1175d1e4f553a5f8db535), thanks.